sports-betting/betting-engine/src/main/java/io/aklivity/zilla/demo/betting/task/SimulatorTask.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2021-2024 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.demo.betting.task;
+
+import static io.aklivity.zilla.demo.betting.EngineContext.MATCHES_TOPIC;
+
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+
+import io.aklivity.zilla.demo.betting.EngineContext;
+
+public class SimulatorTask implements Runnable
+{
+    private static final ObjectMapper mapper = new ObjectMapper();
+    private final EngineContext context;
+
+    public SimulatorTask(
+        EngineContext context)
+    {
+        this.context = context;
+    }
+
+    @Override
+    public void run()
+    {
+        try (KafkaProducer<String, String> producer = new KafkaProducer<>(context.producerProps))
+        {
+            int eventId = 100;
+            List<String[]> teamsList = List.of(
+                new String[]{"Barcelona", "Real Madrid"},
+                new String[]{"Manchester City", "Liverpool"},
+                new String[]{"Bayern Munich", "Borussia Dortmund"},
+                new String[]{"Juventus", "AC Milan"},
+                new String[]{"Dallas Cowboys", "San Francisco 49ers"},
+                new String[]{"Green Bay Packers", "Chicago Bears"},
+                new String[]{"Chicago Bulls", "New York Knicks"},
+                new String[]{"New York Yankees", "Boston Red Sox"}
+            );
+            List<String> sports = List.of("soccer", "football", "hockey", "basketball", "baseball");
+
+            while (!Thread.currentThread().isInterrupted())
```

Comment:
```suggestion
            while (true)
```
If we are interrupted, then an exception is thrown to exit the loop, so no need to also have this condition, agree?